### PR TITLE
loctv fix bug

### PR DESCRIPTION
**Pull request by at-loctang**

### CODE REVIEW

**Comment by dungnh** [comment]
please fix circleCI

File: app/Http/Controllers/Admin/DashboardController.php [hunk 1]
```diff
@@ -14,18 +14,13 @@
...
 {
     /**
-     * Get index
+     * Get index 
      *
      * @return respone
      */
     public function index()
     {
         $title = 'Dashboard';
```
Comment (dungnh) [inline app/Http/Controllers/Admin/DashboardController.php:23]: please don't hard code like this.

**Comment by at-loctang** [comment]
please merge code . I did fix bug ok. Thanks A.Dung.

**Comment by dungnh** [comment]
LGTM. Thank you.

**Pull request merged**